docs/source/en/api/pipelines/stable_diffusion/stable_diffusion_xl.mdx

```diff
@@ -180,17 +265,17 @@ Let's have a look at the image
 
 | Original Image | Ensemble of Denoisers Experts |
 |---|---|
-| ![lion_base](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/lion_base.png) | ![lion_ref](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/lion_refined.png)
+| ![lion_base](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/lion_base_strength.png) | ![lion_ref](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/lion_refined_strength.png)
 
 If we would have just run the base model on the same 40 steps, the image would have been arguably less detailed (e.g. the lion eyes and nose):
 
-<Tip>
-
-The ensemble-of-experts method works well on all available schedulers!
-
-</Tip>
+Note that for the Stable Diffusion XL base and refiner models, the best choice is to use the 
+pipeline described in example 1.) because it works with a multitude of schedulers. Using strength 
+to transition denoising between ensembles is best when using different Stable Diffusion models and
+different schedulers, for example Stable Diffusion XL base for the first expert and a finetuned SD
+1.x model for the refinement expert.
```

Comment:
```suggestion
<Tip>

The ensemble-of-experts method works well on all available schedulers!

</Tip>
```

I would leave this for now. To me it works just fine - it's just different 20% in this case doesn't mean [1-200] but just the final 20% of timesteps which is exactly what we want.